starter/script.js

```diff
@@ -1 +1,176 @@
-// put code here!
+// one hipster's tale
+
+// functions:
+function getAdventure(adventureState) {
+//   input: adventureState, player info
+//   effect: display current adventure
+  console.log(adventureText[adventureState]);
+};
+
+function getResponse(adventureState) {
+//  input: question/response key, playerinfo
+//  returns: option letter
+  getAdventure(adventureState);
+  var a = '';
+  console.log(adventureQ[adventureState]);
+  while (a != 'a' && a != 'b' && a != 'c') {
+    a = prompt(adventureChoices[adventureState]);
+  };
+  if (a === 'a') {
+    player['hipsterIndex'] += 3;
```

Comment:
The dot accessor is typically cleaner and easier to read unless you are using a variable.
```js
player.hipsterIndex += 3;
```
Also see [this stack overflow article](http://stackoverflow.com/questions/4968406/javascript-property-access-dot-notation-vs-brackets#answer-4968448)